packages/cli/src/commonlib/codeFlowLogin.ts

```diff
@@ -181,6 +181,7 @@ export class CodeFlowLogin {
     try {
       await this.startServer(server, serverPort!);
       this.pca!.getAuthCodeUrl(authCodeUrlParameters).then(async (url: string) => {
+        url += "&";
```

Comment:
Docs `#` work as well?
For security, `#` is better than `&` since parameters after `#` are not sent to server.

Reply:
Thanks for the advice, changed to ‘#’.